pkg/operator/ceph/cluster/cluster_external.go

```diff
@@ -173,6 +178,9 @@ func purgeExternalCluster(clientset kubernetes.Interface, namespace string) {
 }
 
 func validateExternalClusterSpec(cluster *cluster) error {
+	if cluster == nil {
```

Comment:
I don't think `cluster` will ever be nil. Did you mean to check for `cluster.Info`?
```suggestion
	if cluster.Info == nil {
```